tensorflow/lite/micro/kernels/xtensa_hifi/xtensa_tf_micro_common.h

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+* Copyright (C) 2019 Cadence Design Systems, Inc.
+*
+* Permission is hereby granted, free of charge, to any person obtaining
+* a copy of this software and associated documentation files (the
+* "Software"), to use this Software with Cadence processor cores only and
```

Comment:
`File 'tensorflow/lite/micro/kernels/xtensa_hifi/xtensa_tf_micro_common.h' failed validation 'Verify match '(Copyright 20[1-9][0-9]((.|
)*)Apache License, Version 2\.0|Licensed to the Apache Software Foundation)''.
  - 1 file(s) failed the validation of Verify match '(Copyright 20[1-9][0-9]((.|
)*)Apache License, Version 2\.0|Licensed to the Apache Software Foundation)'.`